openhtf/io/output/mfg_inspector.py

```diff
@@ -196,7 +198,17 @@ def _ExtractParameters(record, testrun, used_parameter_names):
           else:
             testrun_param.description += '\nValidator: ' + str(validator)
       else:
-        _MangleMeasurement(name, value, measurement, mangled_parameters)
+        attachment = testrun.info_parameters.add()
+        attachment.name = 'multidim_%s' % name
+        dims = [{
+            'uom_suffix': d.uom_suffix.encode('utf8'), 'uom_code': d.uom_code}
+            for d in measurement.dimensions]
+        attachment.value_binary = json.dumps({
+            'outcome': str(testrun_param.status), 'name': name,
+            'dimensions': dims, 'value': value}, sort_keys=True)
+        attachment.type = test_runs_pb2.MULTIDIM_JSON
```

Comment:
hm, if this format is documented somewhere, can you link to it? or reference it somehow?  if it's not, can you put said documentation here?